.circleci/real_config.yml

```diff
@@ -4612,6 +4612,19 @@ workflows:
           devcluster-config: multi-k8s.devcluster.yaml
           run-minikubes: true
 
+      - test-e2e:
+          name: test-e2e-single-k8s
+          context:
+            - dev-ci-cluster-default-user-credentials
+          requires:
+            - build-go-ee
+          parallelism: 1
+          tf2: true
```

Comment:
We may be able to remove this `tf2: true`. I don't think this job needs the pytorch Docker images!